### PR TITLE
auditok: init at 0.2.0

### PR DESCRIPTION
**Pull request by Arguggi**
###### Description of changes

Add [auditok](https://pypi.org/project/auditok/) python package needed for [ffsubsync](https://pypi.org/project/ffsubsync/)

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.05 Release Notes (or backporting 22.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2305-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Review by Flakebi — APPROVED** [review]
Looks fine to me.

Result of `nixpkgs-review pr 209430` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>2 packages built:</summary>
  <ul>
    <li>python310Packages.auditok</li>
    <li>python39Packages.auditok</li>
  </ul>
</details>

**Review by doronbehar — CHANGES_REQUESTED** [review]
2 Comments:

- Did you manage to run the tests from `tests/` directory?
- Commit message should be: `python3Packages.auditok: init at 0.2.0`.

**Comment by Benjamin-L** [comment]
@Arguggi I'm interested in packaging ffsubsync, and could pick this one up if you're no longer working on it.

File: pkgs/development/python-modules/auditok/default.nix
Comment (drupol) [inline pkgs/development/python-modules/auditok/default.nix:16]: Don't inherit `pname`.

Justification: https://github.com/nix-community/nixpkgs-lint/issues/21

File: pkgs/development/python-modules/auditok/default.nix
Comment (drupol) [inline pkgs/development/python-modules/auditok/default.nix:17]:
```suggestion
    hash = "sha256-RsUS4oey4T30gZd0FmrzKyNtdUr75rDchygbIz5a5og=";
```

File: pkgs/development/python-modules/auditok/default.nix
Comment (drupol) [inline pkgs/development/python-modules/auditok/default.nix:24]: There's no need to have 2 lines for this.

**Review by drupol — CHANGES_REQUESTED** [review]
hi !

Thanks for your contribution.

I made some comments.

**Comment by abathur** [comment]
@Benjamin-L Since this is a blocker for something that you and @Arguggi both explicitly mention wanting to get into nixpkgs, my opinion (as a non-committer...) is that it's fair to go ahead and pick this up in a new PR (but it might be a kind gesture to defer to this PR if @Arguggi notices this discussion and starts addressing the feedback from Doron and Pol).

**Comment by doronbehar** [comment]
OP hasn't responded in 3 months for requests by reviewers. Let's close this. They can open a new one when ready.

**Comment by Arguggi** [comment]
Thanks @Benjamin-L  for the updated PR, for some reason github decided to not send any notifications so I missed these comments.

**Comment by Benjamin-L** [comment]
@Arguggi thanks! If you're interested, there's a final PR for ffsubsync here: https://github.com/NixOS/nixpkgs/pull/258465.

**Pull request closed without merging**
